docs/testplan/server-side-copy-testplan.md

```diff
@@ -0,0 +1,53 @@
+|------------------------|--------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|
+| Other Services         | Test Impact on Audits                                              | While audits are running, it should be possible to server side copy with no negative impact on former services                                                                                                                                                          |          |
+|                        | Test Impact on Graceful Exit                                       | While graceful exit is running, it should be possible to server side copy with no negative impact on former services                                                                                                                                                    |          |
+|                        | Test Impact on Garbage Collection                                  | While garbage collection is running, it should be possible to server side copy with no negative impact on former services                                                                                                                                               |          |
+|                        | Test Impact on Data Repair                                         | While data is being repaired, it should be possible to server side copy with no negative impact on former services as data repair should ignore segments with a set ancestor_stream_id                                                                                  |          |
+|                        | Test Impact on Metainfo > Expired Deletion                         | While metainfo loop is going through expired deletion, it should be possible to server side copy with no negative impact on former services                                                                                                                             |          |
+|                        | Test Impact on Accounting/Tally                                    | While accounting services are running, it should be possible to server side copy with no negative impact on former services                                                                                                                                             |          |
+|                        | Test Expiration Dates                                              | For server side copy users are allowed to make new copies of pieces which allows users to change their expiration dates, so in this way users are able to change expiration date of the new object through server side copy                                             |          |
```

Comment:
This won't be possible as currently designed, as storage nodes will delete the pieces on the original expiration date.

Reply:
Updated comment section of testplan with your reasoning, Fadila!